src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const pricePerDay = 40;
+  const firstDiscount = 20;
+  const secondDiscount = 50;
+
+  if (days < 3) {
+    return days * pricePerDay;
+  }
+
+  if (days < 7) {
+    return days * pricePerDay - firstDiscount;
+  }
```

Comment:
Better to save such numbers as `3` and `7` to variables,  now it looks like magic number